nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -745,20 +748,28 @@ in
       }
 
       {
-        assertion = all (conf: with conf;
-          !(addSSL && (onlySSL || enableSSL)) &&
-          !(forceSSL && (onlySSL || enableSSL)) &&
-          !(addSSL && forceSSL)
+        assertion = all (host: with host;
+          count id [ addSSL (onlySSL || enableSSL) forceSSL rejectSSL ] <= 1
```

Comment:
```suggestion
        assertion = all (host: with host; count id [ addSSL (onlySSL || enableSSL) forceSSL rejectSSL ] <= 1
```

Reply:
Actually not sure I agree with this one. Did you really mean to merge the first two lines but not the third?